### ISSUE TITLE
Enhancement: Add Missing Form Labels to Footer

### ISSUE THREAD

**Issue by annakreno** [spacelabdev/spacelab-react#386]
- Newsletter subscription form in Footer does not have a properly associated text labels, so the function/purpose of the form will not be presented to screen reader users.

To resolve, do one of the following:
- Provide an associated label
- Add a descriptive title attribute to the form control
- Reference the label(s) using aria-labelledby. 

For future reference, a properly associated label is:
- a <label> element with a for attribute value that is equal to the id of a unique form control
- a <label> element that surrounds the form control, does not surround any other form controls, and does not reference another element with its for attribute
- a non-empty title attribute, or
- a non-empty aria-labelledby attribute

Corresponding WCAG Standards and Guidelines:
[1.1.1 Non-text Content (Level A)](https://webaim.org/standards/wcag/checklist#sc1.1.1)
[1.3.1 Info and Relationships (Level A)](https://webaim.org/standards/wcag/checklist#sc1.3.1)
[2.4.6 Headings and Labels (Level AA)](https://webaim.org/standards/wcag/checklist#sc2.4.6)
[3.3.2 Labels or Instructions (Level A)](https://webaim.org/standards/wcag/checklist#sc3.3.2)
